recipe/flags/meta.yaml

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
Just in case you didn't see the edit of my comment, the directory is `recipes` not `recipe`. Sorry for the typo in the original comment.